AltServer/Base.lproj/Main.storyboard

```diff
@@ -61,16 +61,18 @@
                         <outlet property="appMenu" destination="uQy-DD-JDr" id="7cY-Ov-AOW"/>
                         <outlet property="authenticationAppleIDTextField" destination="zLd-d8-ghZ" id="wW5-0J-zdq"/>
                         <outlet property="authenticationPasswordTextField" destination="9rp-Vx-rvB" id="ZoC-DI-jzQ"/>
+                        <outlet property="checkForUpdatesMenuItem" destination="Tnq-gD-Eic" id="wIe-Tx-KlM"/>
```

Comment:
I'm guessing this is tied into the earlier stuff. Is this adding a new menu item?